tests/Entity/Post/Comment.php

```diff
@@ -25,6 +26,16 @@ public static function fields()
         ];
     }
 
+    public static function scopes()
+    {
+        return [
+            'yesterday' => function (Query $query) {
+                return $query->where(['date_created :gt' => date('Y-m-d H:i:s', strtotime('yesterday')), 'date_created :lt' => date('Y-m-d H:i:s', strtotime('today'))]);
```

Comment:
You should use `new \DateTIme('yesterday')` instead of `date('Y-m-d H:i:s', strtotime('yesterday'))` here so the database adapter can format the `DateTime` object however it needs to for comparison. Also, it's less typing :).